dataactcore/scripts/initialize.py

```diff
@@ -153,6 +157,9 @@ def main():
     if args.update_domain:
         load_domain_value_files(validator_config_path)
 
+    if args.cfda_load:
```

Comment:
If we're pulling cfda out of `load_domain_value_files`, there needs to be a way for devs to continue loading all the domain values at once, even if it's an argument `-ad` that calls `load_domain_value_files` and then `load_cfda` for now. `-d` should eventually be that thing but if we want to separate for now that would be the best thing